_studio/mfx_lib/shared/src/mfx_vpx_dec_common.cpp

```diff
@@ -406,6 +368,46 @@ namespace MFX_VPX_Utility
         return true;
     }
 
+    bool CheckInfoMFX(const mfxInfoMFX &mfx, mfxU32 codecId, eMFXPlatform platform,  eMFXHWType hwtype)
+    {
+        if (codecId != mfx.CodecId)
```

Comment:
you can do this with ```MFX_CHECK``` that will allow to see what check was failed in logs with enabled tracing